pkg/boot/grub/testdata/bash_wrapper.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Just wondering why we need to build in a bash dependency? Is there any other way?

Reply:
See my big explanation bellow, also the bash scripts are in testdata, so used only for testing.